### PR TITLE
Ignore generated/ instead of ios/

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

File: examples/.gitignore [hunk 1]
```diff
@@ -5,6 +5,6 @@
 .packages
 .pub/
 build/
-ios/
+generated/
```
Comment (devoncarew) [inline examples/.gitignore]: ==> `ios/Generated`?

Comment (tvolkert) [inline examples/.gitignore]: Done.  Was just updating and running ios init to check it :-)

**Comment by devoncarew** [comment]
lgtm!

**Comment by devoncarew** [comment]
Though, we might want to change `'Generated'` to `'generated'`or `'_generated'` or something.

**Comment by tvolkert** [comment]
Looks like we actually generate a .gitignore in ios/, so I just updated that.

@chinmaygarde

**Pull request merged**